components/registry/org.wso2.carbon.registry.indexing/src/main/java/org/wso2/carbon/registry/indexing/AsyncIndexer.java

```diff
@@ -182,6 +183,8 @@ protected void uploadFiles(List<IndexingTask> tasks) throws RegistryException {
     }
 
     protected static class IndexingTask implements Runnable {
+
+        public static final String REGISTRY_RESOURCE_SYMLINK_PATH = "registry.resource.symlink.path";
         private File2Index fileData;
 
         protected IndexingTask(File2Index fileData) {
```

Comment:
Might be better to describe the reason for making this constructor protected. Please add comments.